### ISSUE TITLE
Consider adding a mechanism for python code to explicitly force a reload

### ISSUE THREAD

**Issue by davepeck** [adamchainz/django-browser-reload#244]
### Description

For instance, I'd like to force a reload when I create or update a specific model instance in my app.

Apologies if I simply missed it, but is there an explicitly supported method I can call (or signal I can raise) to achieve this?

**Comment by adamchainz** [comment]
There isn’t a method. Would this be when you run a management command or something? It could be an interesting addition.

**Comment by davepeck** [comment]
Yes, this is in the context of a management command &mdash; in my case, one that's run while I'm also running a local dev server.

(My inelegant solution at the moment is to have the management command overwrite an otherwise unused and empty `reload.html` file sitting in my template hierarchy.)

**Comment by adamchainz** [comment]
This method would need to call [`trigger_reload_soon()`](https://github.com/adamchainz/django-browser-reload/blob/97754e2382b1a28df56332eabce58781f4c953de/src/django_browser_reload/views.py#L49)  *within* the `runserver` process. If you can figure out some way of doing lightweight cross-process communication to do that, it could work as an addition.

But then again, touching a file, as you’re already doing, is a simple cross-process communication method. So it might not be worth adding anything. FYI you can use [`Path.touch()`](https://docs.python.org/3.12/library/pathlib.html#pathlib.Path.touch) as a quick way to do this.

**Comment by davepeck** [comment]
I pondered this over the weekend and eventually decided it probably _isn't_ worth building an explicit mechanism when touching a file will do fine. 

Not sure where you landed &mdash; but if in the same place then maybe a documentation update to suggest this would be useful -- other than that, we can close this!

Thanks again...

**Comment by adamchainz** [comment]
Let's add a documentation section covering touching an existing template.
